package.json

```diff
@@ -18,14 +18,16 @@
   },
   "scripts": {
     "watch": "yarn install-if-deps-outdated && ./node_modules/.bin/gulp watch",
-    "clean-client": "rm -rf _inc/build/ _inc/blocks/ css/",
+    "clean": "yarn clean-client && yarn clean-extensions",
+    "clean-client": "rm -rf _inc/build/ css/",
+    "clean-extensions": "rm -rf _inc/blocks/ ",
     "install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files",
-    "distclean": "rm -rf node_modules",
-    "build": "yarn install-if-deps-outdated && yarn clean-client && yarn build-client && yarn build-extensions",
+    "distclean": "rm -rf node_modules && yarn clean",
```

Comment:
Do we expect `vendor`, `_inc/jetpack-strings.php`, `modules/**/*.min.css.map` and other things [gitignore file](https://github.com/Automattic/jetpack/blob/4a46dd84c2e0680049b44c9d90f6bdc12ed66dec/.gitignore) get cleaned up?